docs/source/debugging.rst

```diff
@@ -189,9 +190,38 @@ Defaults currently look like the following:
      distributed.client: warning
      bokeh: error
 
-So, for example, you could add a line like ``distributed.worker: debug`` to get
+The specific components which you can set are ``distributed.client``,  ``distributed.scheduler``,
+``distributed.nanny``, and ``distributed.worker`` so, for example, you could add a line
+like ``distributed.worker: debug`` to get
 *very* verbose output from the workers.
 
+Furthermore, you can explicitly assign handlers to loggers. The following example
+assigns both file ("output.log") and console output to the scheduler and workers.
+See the `python logging`_ documentation for information on the meaning of
+specific terms here.
+
+.. code-block:: yaml
+
+logging:
+  version: 1
+  handlers:
+    file:
+      class: logging.handlers.RotatingFileHandler
+      filename: output.log
+      level: INFO
+    console:
+      class: logging.StreamHandler
+      level: INFO
+  loggers:
+    distributed.worker:
+      level: INFO
+      handlers: ["file", "console"]
+    distributed.scheduler:
+      level: INFO
+      handlers: ["file", "console"]
```

Comment:
This section should be indented

Reply:
Instead of a json-like list? I thought compact would be nice.

Reply:
I just mean indenting each line one more level so that sphinx renders this `.. code-block:: yaml` section properly. Currently it looks like 

<img width="729" alt="Screen Shot 2020-04-28 at 8 17 12 AM" src="https://user-images.githubusercontent.com/11656932/80491918-f4fadd80-8928-11ea-83ed-ee8a6fc8780a.png">

Reply:
Got it.
I swear this was right before and my ide must have done it.